packages/components/modal/src/Modal.tsx

```diff
@@ -106,6 +126,27 @@ export function Modal({
     topOffset,
   };
```

Comment:
Could you fix this in your PR?

I guess in the past we were doing this weird thing where we destructure the props in the argument of the function and then we assemble everything again in a const called props 🤷 

```jsx
function Component({ children, ...otherProps}) {
  // no const props here
  
  return <div {...otherProps}>{children}</div>
}
```

I've seen this pattern with duplication of props in other components and I really don't know where it came from hahahaha

Reply:
I guess this one is needed here, because `props` are passed to a children function

Reply:
and it's the only place in the modal package where we do this